Dockerfile

```diff
@@ -21,4 +21,9 @@ ENV DATABASE_CONFIG=/var/lib/drone/drone.sqlite
 
 ADD drone_static /drone_static
 
+# Alpine Linux doesn't use pam, which means that there is no /etc/nsswitch.conf,
```

Comment:
me neither, but under certain host configurations the Go program will be unable to resolve domain names without this file. This is because we are using cgo which statically links glibc which looks for this file. Alpine doesn't use glibc (it uses musl) and doesn't include this file by default.

https://github.com/gliderlabs/docker-alpine/issues/11
https://github.com/drone/drone/issues/1269